content/blog/2018/stockfish-10.md

```diff
@@ -8,7 +8,7 @@ aliases:
 
 Official release version of Stockfish 10.
 
-Downloads are available at <http://stockfishchess.org/download/>
+Downloads are available at <https://stockfishchess.org/download/>
```

Comment:
please do not modify old blog posts in this PR

Reply:
Reverted my change as requested.